content/riak/ts/1.3.0/learn-about/bestpractices.md

```diff
@@ -23,20 +23,20 @@ canonical_link: "https://docs.basho.com/riak/ts/latest/learn-about/bestpractices
 There are many ways to interact with and use Riak TS. This page will give recommendations for structuring your storage region (DDL) and choosing your quanta.
 
 
-## DDL Definition
+## TS Table Schema
 
-One of the first things you will encounter setting up Riak TS is defining storage regions, also known as Data Definition Language (DDL). Creating a table will specify the DDL and compile it, which will create a Riak [bucket][glossary bucket]. The bucket information is passed around the ring.
+One of the first things you will encounter in setting up Riak TS is defining a TS table's schema.  You use Data Definition Language (DDL), specifically the `CREATE TABLE` statement, to create a new table. When you create a TS Table, the data structure provided in the `CREATE TABLE` statement is compiled and passed around the Riak TS cluster.
 
-There are two reasons that how you set up your DDL is really important:
+The design of your TS table schema is really important because:
```

Comment:
I hate seeing the words "really" and "thing" in technical writing...

"It is important to pay close attention to the design of your TS table schema because:"
?